File: Ciclo2/1. Reto/Reto1/src/main/java/com/mycompany/reto1/Reto1.java

package com.mycompany.reto1;

import java.util.Scanner;

/**
 * Reto #1
 * @author Caro
 */
public class Reto1 {
    
    public static void main(String[] args){
        Scanner sc = new Scanner (System.in);
        int pfizer = sc.nextInt(); 
        int astraZeneca  = (2 * pfizer) + 4;
        int janssen = (pfizer + astraZeneca ) / 5;
        System.out.println(pfizer + " " + astraZeneca + " " + janssen);
        System.out.println();
        if (janssen >= 0 && janssen <= 20){
            System.out.println("uno");
        }else if(janssen >= 21 && janssen <= 30){
            System.out.println("dos");
        }else if (janssen >= 31 && janssen <= 50){
            System.out.println("tres");
        }else{
            System.out.println("cuatro");
        }
       
    }
}

File: Ciclo1/reto1.py
def puntaje(toledo):
    norato = (2 * toledo) + 4
    barrero = (toledo + norato) // 5
    print(toledo, norato, barrero, end=" ")
    print("\n")
    if barrero >= 0 and barrero <= 20:
        print("uno")
    elif barrero >= 21 and barrero <= 30:
        print("dos")
    elif barrero >= 31 and barrero <= 50:
        print("tres")
    else:
        print("cuatro")

def main():
    toledo = int(input())
    puntaje(toledo)
main()
